deploy/kubernetes/webhook-cm.yaml

```diff
@@ -0,0 +1,11 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: tracee-webhook-config
+data:
+  cfg.yaml: |-
```

Comment:
would be nice to add a comment here pointing to the reference: https://github.com/falcosecurity/falcosidekick/blob/master/config_example.yaml